src/NATS.Client.KeyValueStore/NatsKVStore.cs

```diff
@@ -263,6 +263,19 @@ public async ValueTask<NatsKVEntry<T>> GetEntryAsync<T>(string key, ulong revisi
             NatsDeserializeException? deserializeException = null;
             if (response.Message.Data != null)
             {
+#if NETSTANDARD2_0
+                var bytes = Convert.FromBase64String(response.Message.Data);
+                var buffer = new ReadOnlySequence<byte>(bytes);
+                try
+                {
+                    data = serializer.Deserialize(buffer);
+                }
+                catch (Exception e)
+                {
+                    deserializeException = new NatsDeserializeException(buffer.ToArray(), e);
+                    data = default;
+                }
```

Comment:
Couple of notes here:

1. I think if we can, we should try to keep behavior consistent, even if that means the `Convert.FromBase64String` is done as a try-catch.
2. Is there an issue on the backlog to convert the `StreamMessage.Data` type into something a bit less thrashy? `System.Buffers.Text` [has To/From for utf8 byte sequences](https://learn.microsoft.com/en-us/dotnet/api/system.buffers.text.base64.decodefromutf8?view=net-8.0) and it looks like they work with netstandard20, adapting to such an approach longer term may help consolidate this code as well as lower the deserialization cost of KV and other stream reads in NET8 (as we can hold the base64 in UTF8 bytes rather than UTF16 string, maybe pool in future too? Not sure how easy it would be to do that part though...)

Reply:
(1) 👍 
(2) that sounds good. created one: https://github.com/nats-io/nats.net.v2/issues/518